src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Resources/config/services.yml

```diff
@@ -238,3 +238,10 @@ services:
         arguments:
         - base_uri: "%manage_prod_host%"
           auth: ["%manage_prod_username%", "%manage_prod_password%", "Basic"]
+
+    Surfnet\ServiceProviderDashboard\Application\Console\CleanOldProductionEntitiesCommand:
+        class: Surfnet\ServiceProviderDashboard\Application\Console\CleanOldProductionEntitiesCommand
+        arguments:
+          - '@Surfnet\ServiceProviderDashboard\Infrastructure\DashboardBundle\Repository\EntityRepository'
+          - '@surfnet.manage.client.query_client.prod_environment'
+        tags: ['console.command']
```

Comment:
The command was not available in my console. Can you verify?

https://symfony.com/doc/3.4/console/commands_as_services.html

Reply:
Did you clear your symfony cache?

Reply:
Multiple times, will try it again on my next review round and call you over if I can't get it to work.